src/ui/bot.ts

```diff
@@ -13,6 +13,7 @@ export class YveBotChat {
   constructor(rules: Rule[], opts?: ChatOptions) {
     const DEFAULT_OPTS: ChatOptions = {
       target: 'body',
+      name: 'YveBot',
```

Comment:
I prefer to keep it blank as optional. Just add a new test to ensure the new element.
You don't have to change anymore the example in yvebot page.